frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eclipse-sirius/sirius-components",
-  "version": "0.2.18",
+  "version": "0.2.19",
```

Comment:
Do not upgrade that this way (have a look at the versioning ADR for the proper commands for that and do it in another commit too since it changes a lot of files)

Reply:
Ack